specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-09-01-preview/common/IncidentTypes.json

```diff
@@ -0,0 +1,244 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-09-01-preview",
+    "title": "Common Incident types"
+  },
+  "paths": {},
+  "definitions": {
```

Comment:
Did any of these definitions get updated during the refactoring?  In the future when doing refactoring of specs do all refactoring in a few commits then all changes afterwards to make the diff clean (if that was done please point me to the commit to review from)

Reply:
So sorry for the unorganized refactoring. Will make sure to do as you suggested in my next PRs.
The only change is in `IncidentInfo` which had a an enum with `Critical` value under it's `severity` property. This was a mistake in previous versions as such a severity value doesn't exist for incidents. I referenced the correct `IncidentSeverityEnum` in the severity property instead.